src/devtools/client/debugger/src/components/Editor/Breakpoints/Panel/PanelEditor.js

```diff
@@ -169,12 +165,6 @@ class PanelEditor extends PureComponent {
     const { conditionValue, logValue } = this.state;
     const hasCondition = conditionValue !== null;
 
-    if (!logValue) {
-      // Skip the first render. The state doesn't have the default values for
-      // logValue and conditionValue until after the component mounts.
-      return null;
-    }
```

Comment:
this was the problem, but i think we can get around it by setting the state in the constructor...